.github/workflows/build.yml

```diff
@@ -175,6 +181,24 @@ jobs:
           git-config-name: github-actions[bot]
           git-config-email: github-actions[bot]@users.noreply.github.com
 
+      - name: Host vite-playground under /vite
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
I think we should only run this one on branches that aren't `main`. Can be achieved using an `if` property

Reply:
Okay, will do.

Reply:
Added `if` property in this [commit.](https://github.com/iTwin/iTwinUI/pull/1995/commits/b155bb47f055e02832bd005964c8b24aa27162f3)